.github/workflows/ci-unix-shared-installed.yml

```diff
@@ -29,6 +29,9 @@ jobs:
     - name: Set GCC & G++ compiler (on Linux)
       if: runner.os == 'Linux'
       run: echo "CC=gcc-${{matrix.gcc}}" >> $GITHUB_ENV && echo "CXX=g++-${{matrix.gcc}}" >> $GITHUB_ENV
+    - name: Disable libyuv on macOS
+      if: runner.os == 'macOS'
+      run: echo "CMAKE_AVIF_FLAGS=\"-DAVIF_LIBYUV=OFF\""  >> $GITHUB_ENV
```

Comment:
Add a comment mentioning why it is disabled on macOS

Reply:
Done